dev/sort_qmake_file_entries.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os 
+import glob
+from shutil import copyfile
```

Comment:
Add `from __future__ import print_function` and use `print()` in the rest of the code to be python3 compatible.